cpp/daal/src/services/service_topo.h

```diff
@@ -60,6 +61,9 @@ typedef cpuset_t cpu_set_t;
         #ifdef __x86_64__
             #define LNX_PTR2INT unsigned long long
             #define LNX_MY1CON  1LL
+        #elif __aarch64__
+using LNX_PTR2INT                = unsigned long long;
```

Comment:
Keep the existing style, and use the defines here as well

Reply:
I believe we can just use C's native `uintptr_t` instead